### ISSUE TITLE
Inform screen reader user about follow up command(s)

### ISSUE THREAD

**Issue by rebornix** [microsoft/vscode#186407]
Testing #186218

* Run `/createNotebook load csv to data frame`
* When the response finishes, NVDA starts to read the response
* 🐛 It ignores the follow up command to "Create Notebook"

<img width="354" alt="image" src="https://github.com/microsoft/vscode/assets/876920/b00fa483-36f9-4d75-9980-aba016a2f4a4">

**Comment by rebornix** [comment]
Same for create workspace cc @bhavyaus @joyceerhl 

<img width="352" alt="image" src="https://github.com/microsoft/vscode/assets/876920/fec1e589-d755-4a47-89f0-4089143fe19c">

**Comment by meganrogge** [comment]
This is still a problem. I think we should announce the actions, if any, right after reading the response.

**Comment by meganrogge** [comment]
already happens, /duplicate https://github.com/microsoft/vscode-copilot/issues/278
